frontend/javascripts/viewer/model/bucket_data_handling/pullqueue.ts

```diff
@@ -92,24 +93,33 @@ class PullQueue {
       );
 
       for (const [index, bucketAddress] of batch.entries()) {
-        const bucketBuffer = bucketBuffers[index];
-        const bucket = this.cube.getOrCreateBucket(bucketAddress);
+        try {
+          const bucketBuffer = bucketBuffers[index];
+          const bucket = this.cube.getOrCreateBucket(bucketAddress);
 
-        if (bucket.type !== "data") {
-          continue;
-        }
+          if (bucket.type !== "data") {
+            continue;
+          }
 
-        if (bucketBuffer == null && !renderMissingDataBlack) {
-          bucket.markAsFailed(true);
-        } else {
-          this.handleBucket(bucket, bucketBuffer);
+          if (bucketBuffer == null && !renderMissingDataBlack) {
+            bucket.markAsFailed(true);
```

Comment:
What is the difference between `bucket.markAsFailed(true)` and `failedBucketAddresses.push(bucketAddress)`?